events-api/src/modules/payment/payment.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { PaymentService } from './payment.service';
+import { HttpModule } from '@nestjs/axios';
+
+@Module({
+  providers: [PaymentService],
+  imports: [HttpModule],
```

Comment:
Configure HttpModule here

Reply:
HttpModule.registerAsync({
  imports: [ConfigModule],
  useFactory: async (configService: ConfigService) => ({
    timeout: configService.get('HTTP_TIMEOUT'),
    maxRedirects: configService.get('HTTP_MAX_REDIRECTS'),
  }),
  inject: [ConfigService],
});

Reply:
configured HttpModule in Module file